diff --git a/node_modules/@rosen-chains/evm/dist/EvmChain.js b/node_modules/@rosen-chains/evm/dist/EvmChain.js
index deb16be..7bf2a4c 100644
--- a/node_modules/@rosen-chains/evm/dist/EvmChain.js
+++ b/node_modules/@rosen-chains/evm/dist/EvmChain.js
@@ -96,6 +96,7 @@ class EvmChain extends AbstractChain {
                     data: '0x' + eventId,
                     value: value,
                     chainId: this.CHAIN_ID,
+                    gasLimit: 30000000n,
                 });
             }
             else {
@@ -111,6 +112,7 @@ class EvmChain extends AbstractChain {
                     data: data + eventId,
                     value: 0n,
                     chainId: this.CHAIN_ID,
+                    gasLimit: 30000000n,
                 });
             }
             let estimatedRequiredGas = await this.network.getGasRequired(trx);
