diff --git a/node_modules/@rosen-bridge/evm-address-tx-extractor/dist/extractors/EvmTxExtractor.js b/node_modules/@rosen-bridge/evm-address-tx-extractor/dist/extractors/EvmTxExtractor.js
index 462db32..ba472de 100644
--- a/node_modules/@rosen-bridge/evm-address-tx-extractor/dist/extractors/EvmTxExtractor.js
+++ b/node_modules/@rosen-bridge/evm-address-tx-extractor/dist/extractors/EvmTxExtractor.js
@@ -29,8 +29,7 @@ export class EvmTxExtractor extends AbstractExtractor {
      */
     processTransactions = async (txs, block) => {
         const extractedTxs = [];
-        for (const txRes of txs) {
-            const tx = Transaction.from(txRes);
+        for (const tx of txs) {
             if (tx.from === null) {
                 throw Error('ImpossibleBehavior: RPC transactions must have `from`.');
             }
@@ -40,11 +39,11 @@ export class EvmTxExtractor extends AbstractExtractor {
             if (tx.from.toLowerCase() === this.address) {
                 let status;
                 try {
-                    const result = await txRes.wait(0);
+                    const result = await tx.wait(0);
                     if (result)
                         status = EvmTxStatus.succeed;
                     else
-                        throw Error(`Impossible behavior: Evm Tx [${txRes.hash}] is included in block [${block.hash}] but waiting resulted in null or undefined`);
+                        throw Error(`Impossible behavior: Evm Tx [${tx.hash}] is included in block [${block.hash}] but waiting resulted in null or undefined`);
                 }
                 catch (e) {
                     if (isCallException(e))
@@ -53,7 +52,7 @@ export class EvmTxExtractor extends AbstractExtractor {
                         throw e;
                 }
                 extractedTxs.push({
-                    unsignedHash: tx.unsignedHash,
+                    unsignedHash: Transaction.from(tx).unsignedHash,
                     signedHash: tx.hash,
                     nonce: tx.nonce,
                     address: tx.from.toLowerCase(),
